src/software/ai/hl/stp/tactic/tactic_test.cpp

```diff
@@ -181,60 +189,64 @@ TEST_P(TacticAvoidAreasTest, test_default_avoid_areas)
         refbox_states_str += ss.str();
     }
 
-    EXPECT_EQ(expected_avoid_areas, next_intent->getAreasToAvoid())
+    for (auto constraint : next_intent->getMotionConstraints())
+    {
+        std::cout << constraint << std::endl;
+    }
```

Comment:
Could we remove this? It's easy enough to add printouts back in, but I'd like to keep them  out of committed tests. Alternatively we could add it to the failure output string (like we do with `refbox_states_str`).

Reply:
Yeah left that in by accident

Reply:
removed